components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4960,4 +4963,38 @@ public static String[] extractCredentialsFromAuthzHeader(HttpServletRequest requ
 
         return OAuthUtils.decodeClientAuthenticationHeader(authorizationHeader);
     }
+
+    /**
+     * Check whether TLS client certificate bound access tokens are supported by the server.
+     *
+     * @return     Whether TLS client certificate bound access tokens are supported.
+     */
+    public static boolean isTlsClientCertificateBoundAccessTokensSupported() {
```

Comment:
In the last discussion, we discussed this is not bound to an authenticator and also this should be a binding type. Lets change this with that direction.

Reply:
Removed TLS certificate bound access token related changes in 3c54ef90dce0b0bb1fc707a493b621553251eba8. Will add that configuration after moving towards the discussed direction.